corehq/motech/repeaters/management/commands/generate_repeaters_summary.py

```diff
@@ -0,0 +1,36 @@
+from django.core.management.base import BaseCommand
+from corehq.motech.repeaters.models import RepeatRecord, Repeater
+
+
+class Command(BaseCommand):
+    help = """
+    Shows the number of Repeaters and RepeatRecords per domain.
+    """
+
+    def handle(self, *args, **options):
+        self.stdout.write("\n")
+        self.stdout.write('fetching repeater data...')
+        repeater_summary = Repeater.get_db().view(
+            'repeaters/repeaters',
+            group_level=1,
+            reduce=True
+        ).all()
+        repeaters_by_domain = {info['key'][0]: info['value'] for info in repeater_summary}
+
+        self.stdout.write("\n")
+        self.stdout.write('fetching repeat record data...')
+        repeat_records_summary = RepeatRecord.get_db().view(
+            'repeaters/repeat_records',
+            group_level=1,
+            reduce=True
+        ).all()
+
+        self.stdout.write("\n\n\n")
+        self.stdout.write("Domain\tRepeaters\tRepeatRecords")
+        for info in repeat_records_summary:
```

Comment:
are these sorted in any way? it would be cool if they were in order of size or something similar

Reply:
they aren't sorted. it's easy enough imo to copy/paste the results into an excel/google sheet and sort there...since that's probably where the data will be consumed/shared anyway

Reply:
* correction, these are sorted alphabetically by domain, as couch reduce would do